external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -1655,96 +1664,85 @@ impl<'a> ParsingSymbolicator<'a> {
     ) {
         match mod_use {
             P::ModuleUse::Module(Some(alias_name)) => {
-                let Some(alias_name_start) =
-                    get_start_loc(&alias_name.loc(), self.files, self.file_id_mapping)
+                self.mod_name_symbol(alias_name, mod_defs, mod_ident_str);
+            }
+            P::ModuleUse::Module(None) => (), // nothing more to do
+            P::ModuleUse::Members(v) => {
+                for (name, alias_opt) in v {
+                    self.use_decl_member_symbols(mod_defs, mod_ident_str.clone(), name, alias_opt);
+                }
+            }
+        }
+    }
+
+    /// Get symbols for a module member in the use declaration (can be a struct or a function)
+    fn use_decl_member_symbols(
```

Comment:
This refactoring wasn't strictly necessary, but once done, it arguably makes the code look cleaner.